### ISSUE TITLE
review: abnormallyExpeditedBiologicalProcess

### ISSUE THREAD

**Issue by jseager7** [obophenotype/upheno#492]
This is a proposal for a pattern that is the inverse of '[abnormally delayed biological process](https://github.com/obophenotype/upheno/blob/72f8fe132d5560dba2727c1564e7931f40c4792f/src/patterns/dosdp-dev/abnormallyDelayedBiologicalProcess.yaml)'.

'Expedited' is an antonym of 'delayed', and in this context should be taken to mean a biological process that 'starts _sooner_ than the natural start time or the reference process' (adapting the PATO definition for 'delayed' at [PATO:0000502](https://www.ebi.ac.uk/ols/ontologies/pato/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FPATO_0000502)). This pattern is required for numerous terms in [PHIPO](https://github.com/PHI-base/phipo).

Unfortunately, PATO does not seem to have an inverse for the 'delayed' quality &ndash; nothing is shown on the [OLS page](https://www.ebi.ac.uk/ols/ontologies/pato/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FPATO_0000502), anyway. Presumably PATO would need to add a term with the opposite meaning of 'delayed' before this pattern can be created. I've been told @mah11 can help with this?

Here's the draft:

```yaml
pattern_name: abnormallyExpeditedBiologicalProcess
pattern_iri: http://purl.obolibrary.org/obo/upheno/patterns-dev/abnormallyExpeditedBiologicalProcess.yaml
description: "A biological process which starts sooner than the natural start time or the reference process."

contributors:
  - https://orcid.org/0000-0001-7487-610X

classes:
  expedited: PATO:_______ # no term exists for this quality
  abnormal: PATO:0000460
  biological process: GO:0008150

relations:
  inheres_in: RO:0000052
  has_modifier: RO:0002573
  has_part: BFO:0000051

annotationProperties:
  exact_synonym: oio:hasExactSynonym

vars:
  biological_process: "'biological process'"

name:
  text: "expedited %s"
  vars:
   - biological_process

def:
  text: "Expedited %s."
  vars:
    - biological_process

equivalentTo:
  text: "'has_part' some ('expedited' and ('inheres_in' some %s) and ('has_modifier' some 'abnormal'))"
  vars:
    - biological_process
```

**Comment by matentzn** [comment]
@mah11 @dosumis @ybradford @sbello @chris-grove (and everyone else) for comments. :) Thanks!

**Comment by mah11** [comment]
tl;dr: @jseager7, could you see if PATO:0000694 will do?

As I suspected, dictionaries say that "expedite[d]" can mean either "happens sooner" OR "takes longer"; I'm not aware of any ontology-world usage that constrains "expedite" to the former meaning in a more specialized context.

In FYPO I have used PATO:0000694 'premature' in a de-facto pattern for processes that begin abnormally early. It's is_a PATO:0002325 'onset quality', and defined as "A quality of a process which starts earlier than the natural start time or the reference process. [PATO:LC]".

in principle I'm happy to upgrade the pattern globally to a new PATO class, but at the moment I don't understand what, if any, difference there is between PATO:00003694 and what @jseager7 is requesting here. Perhaps another synonym would suffice (and I'd think the scope should be RELATED because of the ambiguity in the vernacular meaning of "expedite").

**Comment by jseager7** [comment]
@mah11 [PATO:0000694](https://www.ebi.ac.uk/ols/ontologies/pato/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FPATO_0000694) sounds like exactly what we need, thanks. I must have missed the fact that 'premature' existed when making this issue, because the OLS was filtering the tree view to only show 'delayed'.

I agree that 'expedited' doesn't really capture the semantics properly due to the flaws you mentioned. I'll amend the issue to use 'premature' instead of 'expedited', and I'll recommend that we amend PHIPO's terminology to match (since I think PHIPO currently uses 'expedited' in its term labels).

**Comment by mah11** [comment]
@jseager7 
Glad it works for you! I don't see any reason why PATO couldn't add the 'related' synonym anyway, to help with future searching.

**Comment by chris-grove** [comment]
+1 to 'premature' (PATO_0000694), which also has the synonyms 'precocious' and 'early'. We have many 'precocious' terms in the worm phenotype ontology that could use this pattern. Thanks!

**Comment by matentzn** [comment]
@jseager7 can you create a PR for this? Directly into dosdp-dev directory please!

THANKS :)

**Comment by jseager7** [comment]
@matentzn There's now a pull request open at #511.
